design-system/PRESENTATION_PATTERNS.md

```
@@ -0,0 +1,99 @@
+| TMP #5629 unmarked empty tabs *(fixed)* | P5 |
+| Manager-Database #1214 empty default views | P2 |
+| Manager-Database #1215 auth notice in UI | P4 |
+| Inv-Man-Intake #629 non-actionable raw-JSON queue | P6 (+ a real action, app-specific) |
+| Inv-Man-Intake #630 trace-sink + raw filenames | P4, P6 |
+| Portable-Alpha #2021 raw errors / empty states | P2, P3 |
+| Portable-Alpha #2026 upload-only, no sample | P5 |
+| Pension-Data #594 *(deployment program — offline)* | — |
+| LMS #351 root JSON 404 / empty surfaces | P2 (+ deployment) |
+| Default dark theme (TMP/PA/MD/IMI) | P1 |
+
+## Streamlit design kit (most of the fleet is Streamlit)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)